Fitness/Views/Cells/AllGymsCell.swift

```diff
@@ -8,13 +8,30 @@
 import UIKit
 import SnapKit
 
+struct SortedGyms {
+    var openGymsIndices: [Int]
+    var closedGymsIndices: [Int]
+}
+
 class AllGymsCell: UITableViewCell, UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
 
     // MARK: - INITIALIZATION
     static let identifier = Identifiers.allGymsCell
     var collectionView: UICollectionView!
     var gyms: [Gym] = [] {
         didSet {
+            let sortedGyms = sortGyms()
+            let gymsRefenceCopy = gyms
+            gyms = []
```

Comment:
I think we can actually condense this by just doing `gyms.sorted( { isOpen($1) < isOpen($0) })` where `isOpen` is `return (closeTimeToday == "") ? false : (Date() > Date.getDateFromTime(time: openTimeToday)) && (Date() < Date.getDateFromTime(time: closeTimeToday))`